config/config.example.js

```diff
@@ -1,7 +0,0 @@
-module.exports = {
-    appBaseURL: 'http://localhost:3000',
-    port: 3000,
```

Comment:
Why did u delete this file?? Let me explain u the structure. Inside config/ the config.example.js file shud be untouched. When u set up u have to copy it to config/index.js which is ignored using .gitignore and is thus not pushed. Please DO NOT DELETE config.example.js file